unity_buy_android/unity_buy_plugin/src/main/java/com/shopify/unity/buy/UnityBuyPlugin.java

```diff
@@ -1,6 +1,8 @@
 package com.shopify.unity.buy;
 
 public class UnityBuyPlugin {
+    private UnityBuyPlugin() { }
```

Comment:
Even more it must be final. Any Util class must be final, with private constructor.